src/NewsModuleBundle/DataFixtures/ORM/LoadUserData.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace NewsModuleBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\AbstractFixture           as AbstractFixture;
```

Comment:
To wydaje mi się, że jest jakiś dziwny kod StyleCI się do tego dotrzepi, bo PSR (standard pisania kodu)